### PR TITLE
[@types/multicoin-address-validator] Improve types for networkType

### PR DESCRIPTION
**Pull request by alishi973**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://www.npmjs.com/package/multicoin-address-validator#parameters>>

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/multicoin-address-validator/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e81d6d277cb9687e3478111612ce47490fdf4f78
</p>

File: types/multicoin-address-validator/index.d.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export as namespace WAValidator;
 export function validate(address: string, currencyNameOrSymbol?: string, opts?: ValidateOpts | string): boolean;
 export function getCurrencies(): Currency[];
 export function findCurrency(symbol: string): Currency | null;
-
+export type NetworkType = "prod" | "testnet" | "both";
```
Comment (kylemclean) [inline types/multicoin-address-validator/index.d.ts:6]: It looks like at least one other value, `"stagenet"`, is possible.

https://github.com/christsim/multicoin-address-validator/blob/d5f41ba84e0b5e5f7e30a8b303561a68992b8874/src/monero_validator.js#L24

**Review by kylemclean — CHANGES_REQUESTED** [review]

**Review by kylemclean — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
@alishi973 thanks!

**Comment by alishi973** [comment]
Ready to merge

**Pull request merged**